eventually/vote/models.py

```diff
@@ -36,17 +41,31 @@ class Vote(models.Model):
     """
 
     MULTI_CHOICES = (
-        ('O', 'One'),
-        ('M', 'Multi'),
+        ("O", 'One'),
+        ("M", 'Multi'),
     )
 
+    event = models.ForeignKey(Event, on_delete=models.CASCADE, null=True)
     is_active = models.BooleanField(default=True)
     is_extended = models.BooleanField(default=True)
     title = models.TextField(max_length=100)
```

Comment:
use CharField insted of TextField

Reply:
if we have (example) url: team/7/event/5/vote/4 maybe you need to create a ForeignKey to Team object to check the correct url route.